homeassistant/components/zha/device_tracker.py

```diff
@@ -103,3 +106,19 @@ def battery_level(self):
         Percentage from 0-100.
         """
         return self._battery_level
+
+    @property
+    def device_info(  # pylint: disable=overridden-final-method
+        self,
+    ) -> DeviceInfo | None:
+        """Return device info."""
+        # We opt ZHA device tracker back into overriding this method because
+        # it doesn't track IP-based devices.
+        # Call Super because ScannerEntity overrode it.
+        return super(ZhaEntity, self).device_info
```

Comment:
It's probably better and safer to invert the inheritance in this class and put `ScannerEntity` last. That's our standard.